hack/config-default.sh

```diff
@@ -12,6 +12,9 @@ image_pull_policy=${IMAGE_PULL_POLICY:-IfNotPresent}
 verbosity=${VERBOSITY:-2}
 package_name=${PACKAGE_NAME:-kubevirt-dev}
 kubevirtci_git_hash="2104200713-f58c002"
+conn_check_ipv4_address=${CONN_CHECK_IPV4_ADDRESS}
```

Comment:
I think worth to give it a default value (`""`)
same for the ipv6 one and dns

Reply:
If I understand what you have meant , the defaults are passed in the code. For exapmle here: 
https://github.com/kubevirt/kubevirt/blob/98902fcfc82f0f1f33d99869d7315d69fa11ec8a/tests/network/vmi_networking.go#L735
The default is 8.8.8.8

Reply:
The code has default value in case the command line argument is `""`
i mean here in the bash to do something like
conn_check_ipv4_address=${CONN_CHECK_IPV4_ADDRESS:-""}
so it will be explicit that if the var is not set its fine and gives the `""` value

btw, otherwise for example if `set -u` will be used it will fail i think
because it use an unset variable

Reply:
I understand. Thanks for explaining. Done.